wisdem/rotorse/rotor_power.py

```diff
@@ -648,7 +648,7 @@ def constr_Tmax(pitch_i, Uhub_i, Omega_rpm_i):
             # fix_pitch_regI12, which locks pitch in region I 1/2, is on
             if (
                 ((Omega[i] == Omega_tsr[i]) and not self.peak_thrust_shaving)
-                or ((Omega[i] == Omega_tsr[i]) and self.peak_thrust_shaving and (T[i] <= max_T))
+                or ((Omega[i] == Omega_tsr[i]) and self.peak_thrust_shaving and (T[i]/max_T <= 1.04))
```

Comment:
maybe I don't understand 1.04?

Reply:
Eliot had a point on the Region II power curve at optimal TSR-rpm that was 0.1% above the thrust shaving target and the code went through the pitch optimization sequence, creating the plot we saw.  When I relaxed the tolerance in that if statement, things looked better.  I felt comfortable with the relaxation because the thrust shaving compliance for all points in Region 2.5 was only accurate to ~5% coming out of our inner scipy optimization loops.

Given that Eliot's issue was that blade twist needed more work, I was on the fence about whether to push this change.  I would be okay reverting it if that is your preference.  I should probably at least add a code comment. . .